src/tests/ftest/osa/osa_online_drain.yaml

```diff
@@ -54,11 +54,11 @@ container:
     properties: cksum:crc16,cksum_size:16384,srv_cksum:on
     oclass: RP_2G1
 ior:
-    no_parallel_job: 10
+    no_parallel_job: 1
     client_processes:
       np: 2
     test_file: /testFile
-    repetitions: 1
+    repetitions: 5
```

Comment:
Good question.   With parallel jobs, I was seeing more dfs_write issues for which I created DAOS-6537. I may enable parallel jobs later.... Regarding, repetitions, it is increased so that we have IO transactions for the entire test run (ie: during drain, exclude/reintegrate, extend).

Reply:
In that case we should note somewhere to re-enable tests with parallel io. Not sure if the new feature of CANCEL_FOR_TICKET can be application here. But may be Phil can tell more about that.

Reply:
I think CANCEL_FOR_TICKET will not work here. Let me see whether we resolve issues with single IO thread. Once resolved, I will gradually increment the parallel io.

Reply:
Added 2 parallel jobs for testing...

Reply:
parallel job added now.